runtime/doc/filetype.txt

```diff
@@ -510,63 +510,82 @@ Local mappings:
 	to the end of the file in Normal mode.  This means "> " is inserted in
 	each line.
 
+
 MAN					*ft-man-plugin* *:Man* *man.vim*
 
-View manpages in Nvim. Supports highlighting, completion, locales, and
-navigation. Also see |find-manpage|.
+View man pages in Neovim.  Supports highlighting, completion, locales, and
```

Comment:
`Nvim` is used throughout the docs, see the header of this file.

Reply:
Consider it done.